.github/workflows/buld_publish_docs.yml

```diff
@@ -0,0 +1,44 @@
+name: Deploy to GitHub Pages
```

Comment:
I believe the workflow file should be named `.github/workflows/build_publish_docs.yml`.

Reply:
Good catch @huynguyen-hl. The file name has been updated.